sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -352,10 +352,15 @@ function getUserPropertiesOrUndefined(value: any): { [key: string]: any } | unde
     return undefined;
   }
   const properties: any = {};
-  const rawProperties = value[keyValuePairXMLTag];
+  let rawProperties;
+  if (!Array.isArray(value[keyValuePairXMLTag]) && value[keyValuePairXMLTag]?.Key) {
+    // When a single property is present
```

Comment:
From just this context it's not clear why this has to be array (clearly you want it to be). Is it more obvious if I look at more of the code or should this comment be expanded?

Reply:
I thought it would be clear from the code, added more comments anyway :)